jen/group.go

```diff
@@ -118,8 +118,22 @@ func (g *Group) renderItems(f *File, w io.Writer) (isNull bool, err error) {
 // Render renders the Group to the provided writer.
 func (g *Group) Render(writer io.Writer) error {
 	f := NewFile("")
+	return g.RenderWithFile(writer, f)
+}
+
+// GoString renders the Group for testing. Any error will cause a panic.
+func (g *Group) GoString() string {
+	buf := bytes.Buffer{}
+	if err := g.Render(&buf); err != nil {
+		panic(err)
+	}
+	return buf.String()
+}
+
+// RenderWithFile renders the Statement to the provided writer
```

Comment:
`Statement` should be `Group` in this comment. 

Also, let's change the comment to "RenderWithFile renders the Group to the provided writer, using imports from the provided file."

Reply:
Fixed, thanks